network/main.tf

```diff
@@ -0,0 +1,139 @@
+# --- network/main.tf ---
+
+data "aws_availability_zones" "available" {}
+
+resource "random_integer" "random" {
```

Comment:
This resource is no longer used. It should be deleted

Reply:
right, yes.